pyproject.toml

```diff
@@ -40,14 +37,18 @@ tqdm = ">=4.62.3"
 xarray = ">=0.20.2"
 joblib = ">=1.2.0"
 pyarrow = ">=14.0.1"
+fastparquet = ">=0.8.0"
+ipympl = {version = ">=0.9.1", optional = true}
+ipywidgets = {version = "^7.7.1", optional = true}
```

Comment:
These are required by the interactive calibration tools even without jupyter environment, so they cannot be optional.

Reply:
I see. I didn't know they worked otherwise because their documentation states them working with jupyter notebooks

Reply:
Not sure how much sense they make without jupyter, but the whole module cannot be loaded if they are not there, so they definitely need to be there. Maybe we can change this in the future.